closure/goog/ui/emoji/emojipicker_test.js

```diff
@@ -810,6 +810,31 @@ testSuite({
     picker.dispose();
   },
 
+  testGetSelectedEmoji_urlPrefix() {
+    const defaultImg = 'http://example.com/../../demos/emoji/none.gif';
+    const picker = new EmojiPicker(defaultImg);
+    picker.setDelayedLoad(false);
+    picker.addEmojiGroup(emojiGroup1[0], emojiGroup1[1]);
+    picker.setUrlPrefix('http://example.com/')
```

Comment:
This URL prefix may not work, as that file doesn't exist. I'd recommend instead using a prefix that is really just adding useless pieces to the path, e.g. 'a/b/../../' that is in effect a no-op when loaded by the browser but will cause the underlying URL to look different.